internal/engine/local/controller.go

```diff
@@ -215,17 +215,31 @@ func processStatuses(
 	statusCh chan statusAndMetadata,
 	st store.RStore,
 	manifestName model.ManifestName,
-	stillHasSameProcNum func() bool) {
+	proc *currentProcess) {
+
+	stillHasSameProcNum := proc.stillHasSameProcNum()
+
 	for sm := range statusCh {
-		if !stillHasSameProcNum() {
+		if !stillHasSameProcNum() || sm.status == Unknown {
 			continue
 		}
-		procState := sm.status.ToProcessState()
-		if procState != "" {
+
+		var runtimeStatus model.RuntimeStatus
+		if sm.status == Done || sm.status == Error {
+			runtimeStatus = model.RuntimeStatusError
```

Comment:
this seem wrong for jobs - `sm.Status == Done` doesn't indicate an error, right?

Reply:
This was an unintentional change in behavior, will remove it.

For context, previously [`ToRuntimeStatus()`](https://github.com/tilt-dev/tilt/blob/v0.18.3/internal/engine/local/controller.go#L241-L259) would just return an empty str for `Done`, so no update would get emitted, so we can continue to do the same thing here.

Reply:
Restored parity with previous behavior in 737a56b